rules/windows/initial_access_dns_server_overflow.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+A specially crafted DNS SIG request can manipulate a known vulnerability in some Windows DNS servers which result in
+Remote Code Execution (RCE) or a Denial of Service (DoS) from crashing the service.
+"""
+false_positives = [
+    """
+    Environments that leverage DNS responses over 60k bytes will identify false positives.
+    """,
+]
+index = ["packetbeat-*", "filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Malicious DNS SIG Request"
+note = "This network traffic has been observed exploiting CVE-2020-1350 (SIGRed)"
+references = [
+    "https://research.checkpoint.com/2020/resolving-your-way-into-domain-admin-exploiting-a-17-year-old-bug-in-windows-dns-servers/",
+    "https://msrc-blog.microsoft.com/2020/07/14/july-2020-security-update-cve-2020-1350-vulnerability-in-windows-domain-name-system-dns-server/",
+    "https://github.com/maxpl0it/CVE-2020-1350-DoS",
+]
+risk_score = 73
+rule_id = "11013227-0301-4a8c-b150-4db924484475"
+severity = "medium"
```

Comment:
The other two are "high".  Just double-checking this is intended.

Reply:
Yep intentional - the other two indicate some form of control on the system, whereas this only reflects a potential attempt

Reply:
Typically if we do medium severity we adjust the risk score to 47.